### PR TITLE
Adds a config-optional endgame chat message

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This basically does what we do for roundstart announcements, but for round end.

With a delay between a round ending, the server rebooting, and a new round starting, sometimes it feels like players would be more likely to catch a roundstart when they know the previous game has just ended, and not a few moments before the next one starts.

## Why It's Good For The Game

This idea was suggested to me several times by many people who don't have good connections to servers and keep missing roundstart because they just aren't given enough time to get on SS13.

I also included Round ID in this, so people who know what time they've played a round can also now easily see which round it was, if they wanted to go back to look at the logs for any reason they have.

## Changelog

:cl:
config: There's now a config-optional announcer for a round ending.
/:cl:

### CODE REVIEW

**Review by MrsTonedOne — APPROVED** [review]

**Comment by Mothblocks** [comment]
Why does this need to be a config?

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -238,6 +238,7 @@
 	//Set news report and mode result
 	mode.set_round_result()
 
+	send2chat("Round [GLOB.round_id ? GLOB.round_id : "NULL"] just ended.", CONFIG_GET(string/chat_announce_end_game))
```
Comment (Mothblocks) [inline code/__HELPERS/roundend.dm:241]:
```suggestion
	send2chat("[GLOB.round_id ? "Round [GLOB.round_id]" : "The round has"] just ended.", CONFIG_GET(string/chat_announce_end_game))
```

**Comment by Mothblocks** [comment]
Oh I am dumb, it's a string

File: config/config.txt [hunk 2]
```diff
@@ -492,6 +492,9 @@ MINUTE_CLICK_LIMIT 400
 ## Send a message with the station name starting a new game.
 #CHAT_ANNOUNCE_NEW_GAME
 
+## Send a message with the round ID when a game has just ended.
```
Comment (Mothblocks) [inline config/config.txt:495]: I don't love the wording on this, this makes it sound like a flag and not a string, especially with the name of the config.

Can you write a better description and potentially rename the config entry?

Comment (JohnFulpWillard) [reply config/config.txt:495]: I renamed both of them

**Review by spookydonut — APPROVED** [review]

**Pull request merged**